src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 40 * days;
```

Comment:
Move all the magic numbers 40, 50, 20 to constants

Reply:
magic numbers 😄